packages/server/test/e2e/3_issue_2891_spec.coffee

```diff
@@ -0,0 +1,15 @@
+e2e = require("../support/helpers/e2e")
+Fixtures = require("../support/helpers/fixtures")
+
+describe "e2e issue 2891", ->
+  e2e.setup()
+
+  ## https://github.com/cypress-io/cypress/issues/2891
+
+  it "passes", ->
+    e2e.exec(@, {
+      project: Fixtures.projectPath('default-layout')
```

Comment:
Another style thing. Always use double quotes in CoffeeScript for consistency's sake. Some day we'll have all this converted to JS and properly linted 😅